ibm_vpd_utils.hpp

```diff
@@ -567,5 +567,15 @@ void findBackupVPDPaths(std::string& backupEepromPath,
  */
 void getBackupRecordKeyword(std::string& record, std::string& keyword);
 
+/**
+ * @brief Check if given FRU is read only or not.
+ *
+ * @param[in] vpdPath - VPD path of the FRU.
+ * @param[in] jsObject - Inventory JSON object.
+ *
+ * @return 0 if FRU is not read only. 1 if FRU is read only. -1 on processing
```

Comment:
this should be boolean , any API which is a question should return only. true or false.

Reply:
done